cmake/adjust/template/vendor/C_Clang.cmake

```diff
@@ -0,0 +1,51 @@
+include_guard(GLOBAL)
+
+# Debug
+set(NBL_C_DEBUG_COMPILE_OPTIONS
+	-ggdb3 -Wall -fno-omit-frame-pointer -fstack-protector-strong
+)
+
+# Release
+set(NBL_C_RELEASE_COMPILE_OPTIONS
+	-fexpensive-optimizations
+)
+
+# RelWithDebInfo
+set(NBL_C_RELWITHDEBINFO_COMPILE_OPTIONS "")
+
+# Global
+list(APPEND NBL_C_COMPILE_OPTIONS
+	-Wextra
+	-fno-strict-aliasing
+	-msse4.2
+	-maes
+	-mfpmath=sse		
+	-Wextra
+	-Wno-sequence-point
+	-Wno-unused-parameter
+	-Wno-unused-but-set-parameter
+	-Wno-c++98-compat
+	-Wno-c++98-compat-pedantic
+	-Wno-padded
+	-Wno-unsafe-buffer-usage
+	-Wno-switch-enum
+	-Wno-error=ignored-attributes
+	-Wno-error=unused-function
+	-Wno-error=unused-variable
+	-Wno-error=unused-parameter
+	-Wno-error=ignored-attributes
+	-Wno-error=non-pod-varargs
+	-fno-exceptions
+)
+
+if(NBL_SANITIZE_ADDRESS)
+	list(APPEND NBL_C_COMPILE_OPTIONS -fsanitize=address)
+endif()
+
+if(NBL_SANITIZE_THREAD)
+	list(APPEND NBL_C_COMPILE_OPTIONS -fsanitize=thread)
+endif()
+
+# our pervious flags-set function called this, does not affect flags nor configs so I will keep it here temporary
+# TODO: move it out from the profile
+link_libraries(-fuse-ld=gold)
```

Comment:
same comments as for C++, maybe unify a bit with a common cmake include ?

Reply:
resolved